pkg/controller/installation/bootstrapReconciler.go

```diff
@@ -310,54 +310,31 @@ func (r *Reconciler) reconcilerGithubOauthSecret(ctx context.Context, serverClie
 }
 
 func (r *Reconciler) reconcileRHMIConfigPermissions(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
-	configRole := &rbacv1.Role{
+	configRoleBinding := &rbacv1.RoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "rhmiconfig-dedicated-admins-role",
+			Name:      "rhmiconfig-dedicated-admins-role-binding",
 			Namespace: r.ConfigManager.GetOperatorNamespace(),
 		},
 	}
 
-	if _, err := controllerutil.CreateOrUpdate(ctx, serverClient, configRole, func() error {
-		configRole.Rules = []rbacv1.PolicyRule{
-			{
-				APIGroups: []string{"integreatly.org"},
-				Resources: []string{"rhmiconfigs"},
-				Verbs:     []string{"update", "get", "list", "watch"},
-			},
-			{
```

Comment:
does removing this role also remove the dedicated-admins ability to watch, get, lost on the RHMI CR? I'm not sure if that's a problem or not.

Reply:
@maleck13 do we still need the dedicated-admins to have this permission, or is deleting both acceptable?

Reply:
It does, but when installing through OLM it performs all these role aggregations that grant them viewing permissions, so effectively they still can view the RHMIConfig

Reply:
Oh sorry I see now you meant RHMI. I'll double check, but I think it's the same, as RHMI is owned by the operator, so OLM will aggregate these roles for the RHMI as well

Reply:
ah ok, then I think we're fine if that is true.

Reply:
I ran this locally, as per above steps, and list priv on RHMI was removed from the dedicated-admin. Is this ok? @philbrookes @sergioifg94

Reply:
@briangallagher I think the best way to verify the effective permissions is to deploy RHMI through the add-on, then change the CSV to point to an image generated from this PR, and log in as dedicated admin. The reason is that OLM adds some permissions to dedicated admins over the CRs owned by the operators, so it will vary from the permissions you'll see when running locally

Reply:
@sergioifg94 Thanks, confirming that worked. I'll assume we are ok with the local scenario.